tests/test_meta_functions_against_registry.py

```diff
@@ -43,6 +43,30 @@ def test_get_lp_token(metaregistry, registries, max_pools):
             assert actual_output == metaregistry_output
 
 
+def test_get_pool_from_lp_token(metaregistry, registries, max_pools, curve_api):
+
+    for i, registry in enumerate(registries):
+        total_pools = registry.pool_count() if max_pools == 0 else max_pools
+        for pool_index in range(total_pools):
+            metaregistry_output = None
+            pool = registry.pool_list(pool_index)
+
+            # get_lp_token
+            if (
```

Comment:
Perhaps just a stylistic thing that bugs me, but the fact this keeps reappearing and is so long it needs to be on two lines makes me think it could be kicked to a separate function so it need only be written once

Reply:
Amended with 

```
if x in [abc, def] : 
    # do something
```

Reply:
Could do it, but I think its a stylistic choice. To me it is readable if instead we do the following (in the new version this is how its done):

```
if x in [a, b, x]:
    # do something
```